protractor_spec.js

```diff
@@ -18,7 +18,8 @@ var IGNORED_TESTS = [
   'closure/goog/net/jsloader_test.html',
   'closure/goog/net/filedownloader_test.html',
   'closure/goog/promise/promise_test.html',
-  'closure/goog/editor/plugins/abstractdialogplugin_test.html'
+  'closure/goog/editor/plugins/abstractdialogplugin_test.html',
```

Comment:
They're the tests that have promise timeouts that fail before the set timeout value. I just add them to this list as they come up.